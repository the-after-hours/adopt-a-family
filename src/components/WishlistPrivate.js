// Page to create/view wishlists - Private view

import React from 'react';

const WishlistPrivate = (props) => (
  <div>
    <h1>Wishlist Private page</h1>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
      commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
      est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed
      do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
      minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
      ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
      est laborum.
    </p>
    <ul>
      <li>list element 1</li>
      <li>list element 2</li>
      <li>list element 3</li>
      <li>list element 4</li>
    </ul>
  </div>
);

export default WishlistPrivate;
