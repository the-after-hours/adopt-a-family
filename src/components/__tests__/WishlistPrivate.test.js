import React from 'react';
import WishlistPrivate from '../WishlistPrivate';
import renderer from 'react-test-renderer';

describe('WishlistPrivate', () => {
  test('WishlistPrivate snapshot', () => {
    const component = renderer.create(<WishlistPrivate />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
