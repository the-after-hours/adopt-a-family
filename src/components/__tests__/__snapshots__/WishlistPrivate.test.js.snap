// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WishlistPrivate WishlistPrivate snapshot 1`] = `
<div>
  <h1>
    WishlistPrivate page
  </h1>
</div>
`;
